src/style.css

```diff
@@ -0,0 +1,85 @@
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: 'Roboto', sans-serif;
+  font-size: 16px;
+  color: #333;
+  background: #f5f5f5;
+  width: 100%;
+  margin: 60px 270px;
+}
+
+.todo {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.top,
+.add {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 50%;
+  height: 30px;
+  padding: 0 20px;
+  background: #fff;
+  border: 1px solid #e6e6e6;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+}
+
+.add input {
+  width: 100%;
+  height: 100%;
+  border: none;
+  outline: none;
+  font-size: 16px;
+  font-family: 'Roboto', sans-serif;
+  color: #333;
+}
+
+.tskelement {
+  list-style-type: none;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 50%;
+  height: 30px;
+  padding: 0 20px;
+  background: #fff;
+  border: 1px solid #e6e6e6;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+}
+
+.btnCompleted {
+  width: 50%;
+  height: 40px;
+  padding: 0 20px;
+  background: rgb(247, 245, 245);
+  border: 1px solid #e6e6e6;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+  cursor: pointer;
+}
+
+svg {
+  width: 15px;
+  height: 15px;
+  color: #333;
+  fill: rgb(201, 198, 198);
+  cursor: pointer;
+}
+
+.tasks svg {
+  width: 20px;
+  height: 20px;
+}
+
+/* .hide {
+  display: none;
+} */
```

Comment:
- [ ] Please, consider removing not using code - it will make your code professional 👍